pkg/virt-controller/watch/vm_test.go

```diff
@@ -175,6 +175,61 @@ var _ = Describe("VirtualMachine", func() {
 			testutils.ExpectEvent(recorder, SuccessfulDataVolumeCreateReason)
 		})
 
+		It("should not failed DataVolume for VirtualMachineInstance until after timeout", func() {
```

Comment:
maybe this can be better `should not try to delete VirtualMachineInstance DataVolume before timeout`